tools/harness-automation/autothreadharness/pdu_controller.py

```diff
@@ -189,6 +196,218 @@ def close(self):
         pass
 
 
+class EatonPduController(PduController):
+
+    outlet_oid_cmd_get_state_base = '1.3.6.1.4.1.534.6.6.7.6.6.1.2.0.'
+    outlet_oid_cmd_set_on_base = '1.3.6.1.4.1.534.6.6.7.6.6.1.4.0.'
+    outlet_oid_cmd_set_off_base = '1.3.6.1.4.1.534.6.6.7.6.6.1.3.0.'
+    outlet_oid_cmd_reboot_base = '1.3.6.1.4.1.534.6.6.7.6.6.1.5.0.'
+    outlet_oid_cmd_set_reboot_delay_seconds_base = '1.3.6.1.4.1.534.6.6.7.6.6.1.8.0.'
+
+    PDU_COMMAND_TIMEOUT = 5
+
+    def open(self, **params):
+        missing_fields = ['ip', 'port']
+        missing_fields = [
+            field for field in missing_fields if field not in params.keys()
+        ]
+        if missing_fields:
+            raise KeyError(
+                "Missing keys in PDU params: {}".format(missing_fields))
```

Comment:
Suggest to use single quotes to quote strings in this and followings for being consistent with the previous program format
```suggestion
                'Missing keys in PDU params: {}'.format(missing_fields))
```

Reply:
Done